pkgs/by-name/gi/git-town/package.nix

```diff
@@ -13,7 +13,7 @@
 
```

Comment:
Commit title nitpick (thanks GitHub for still not having the option to do that):

nixpkgs does not use conventional commits, in particular the part before `:` is used to decide which tests to run (so currently it tries to run tests for `
chore(git-town)`, which does not exist).